### ISSUE TITLE
Enabling any artifacts make the lunar cleansing pool always drop irradiant pearls

### ISSUE THREAD

**Issue by Durbin-Elijah** [ImKyle4815/ArtifactEnsemble#1]
Here is a youtube video showing what happens: https://youtu.be/VpZj3a6m5sA

Turning on any of the artifacts with this mod seems to make this happen but I haven't tested them all individually.

Having the mod installed but not enabling any artifacts does not cause this issue.

**Comment by ImKyle4815** [comment]
That’s strange… when I made the mod this never occurred, though I haven’t played for a few months so maybe an update broke it. I’ll try and take a look soon.

Have you tried this without any other mods loaded?

**Comment by Durbin-Elijah** [comment]
I haven't, since trying to get a cleansing pool to spawn without the spawn chance modifier is a pain, however I did note that it wasn't happening without using one of the artifacts, also it happened without using the spawn chance modifier mod (which is what prompted me to test it).

I think the pearl is being removed from the spawn pool somehow, since I went in with command + bazaar and the only command option was irradiant pearl. That could also be because of command weirdness (since cleansing pools aren't supposed to be possible with command) but it's something.

The only other thing I can think of is it being a vanilla bug where cleansing pools have a chance of only dropping irradiant pearls somehow, but I think someone would have noticed that sooner. I can try running a few times with no mods besides the chance increase which would probably narrow it down.  Trying to do it fully vanilla would be a huge pain though.
